### PR TITLE
Modifications for multilingualization

### PR DESCRIPTION
**Pull request by yamamoto-ryuzo**
I think it will be OK this time.

### CODE REVIEW

**Comment by pka** [comment]
@yamamoto-ryuzo thanks a lot for your PR! I had a look at the changes and some translations changing from "obsolete" to "unfinished" look a bit suspicious to me. I'll ask @manisandro to have another look at the PR.

File: InstantPrintPlugin.py [hunk 1]
```diff
@@ -24,16 +24,19 @@ def __init__(self, iface):
...
+        
         # Localize
         locale = QSettings().value("locale/userLocale")[0:2]
         localePath = os.path.join(self.pluginDir, 'i18n', 'instantprint_{}.qm'.format(locale))
 
         if os.path.exists(localePath):
             self.translator = QTranslator()
             self.translator.load(localePath)
-            QCoreApplication.installTranslator(self.translator)
+            QCoreApplication.installTranslator(self.translator)
```
Comment (manisandro) [inline InstantPrintPlugin.py:35]: Please avoid whitespace changes

**Comment by manisandro** [comment]
Yes, just push follow-up commits to your fork with the requested changes, and then the PR will update itself automatically.

**Comment by yamamoto-ryuzo** [comment]
Is this correct?

**Comment by yamamoto-ryuzo** [comment]
changing from "obsolete" to "unfinished" 
↑　By the way, how can I do the above?

↓　It is auto-generated.
image

**Comment by manisandro** [comment]
If you open the ts file in qt linguist, you can mark the translaton as finished:
image

**Comment by yamamoto-ryuzo** [comment]
I marked the Japanese.
Should I mark other translations as well?
I have not been able to confirm the translation.
image

**Comment by yamamoto-ryuzo** [comment]
What else should I do?

File: i18n/instantprint_de.ts [hunk 2]
```diff
@@ -132,4 +136,27 @@ Keine Sonderzeichen erlaubt</translation>
...
+    </message>
+    <message>
+        <location filename="../ui/ui_printdialog.py" line="80"/>
+        <source>Layout:</source>
+        <translation type="unfinished">Zusammenstellung:</translation>
+    </message>
+    <message>
+        <location filename="../ui/ui_printdialog.py" line="81"/>
+        <source>Scale:</source>
+        <translation type="unfinished">Skala:</translation>
```
Comment (manisandro) [inline i18n/instantprint_de.ts:154]: These should be marked as finished

**Comment by manisandro** [comment]
Thanks!

**Comment by yamamoto-ryuzo** [comment]
I should have simply marked it as complete.
Thank you very much.

**Pull request merged**